src/FakeClient/FakeCaptureEventProcessor.h

```diff
@@ -41,16 +42,18 @@ class FakeCaptureEventProcessor : public orbit_capture_client::CaptureEventProce
   ~FakeCaptureEventProcessor() override {
     {
       LOG("Events received: %u", event_count_);
-      ErrorMessageOr<void> event_count_write_result =
-          orbit_base::WriteStringToFile(kEventCountFilename, std::to_string(event_count_));
+      std::filesystem::path filepath(absl::GetFlag(FLAGS_output_path));
+      ErrorMessageOr<void> event_count_write_result = orbit_base::WriteStringToFile(
+          filepath.append(kEventCountFilename), std::to_string(event_count_));
       FAIL_IF(event_count_write_result.has_error(), "Writing to \"%s\": %s", kEventCountFilename,
               event_count_write_result.error().message());
     }
 
     {
       LOG("Bytes received: %u", byte_count_);
-      ErrorMessageOr<void> byte_count_write_result =
-          orbit_base::WriteStringToFile(kByteCountFilename, std::to_string(byte_count_));
+      std::filesystem::path file_path(absl::GetFlag(FLAGS_output_path));
+      ErrorMessageOr<void> byte_count_write_result = orbit_base::WriteStringToFile(
+          file_path.append(kByteCountFilename), std::to_string(byte_count_));
```

Comment:
Maybe just declare a single `std::filesystem::path file_path{absl::GetFlag(FLAGS_output_path)};` at the top of the destructor, and use `operator/` instead of `.append` where it's used.

Reply:
done